### ISSUE TITLE
GameGuru Max Request/Suggestion - Leaning

### ISSUE THREAD

**Issue by Lemuripest93** [TheGameCreators/GameGuruRepo#3080]
I believe this has already been requested by another user, however making your character lean left and right (with the Q and E key respectively) would be super useful, however unlike the other user's request, i'd like to be able to lean _without_ a weapon as well, this would make games with an escape motive more suspenseful.

Leaning should be toggleable under the player settings, as not every game will need this, this also doesn't require any changes to animations (for the time being), as it functions similar to a camera tilt essentially, obviously feel free to alter or play around with this idea as you see fit.

**Comment by MonkeyFrogStudio** [comment]
Ideas like this one would work great with my suggestion here:

https://github.com/TheGameCreators/GameGuruRepo/issues/2916

With Player Behaviors (or something like them), then developers could simply drag-n-drop whatever player abilities they wanted their players to have onto the player, totally customizing their player, including things like "leaning".

**Comment by Lemuripest93** [comment]
Actually i think that's quite an impressive and exciting idea, and would work excellently with one of my planned projects for when the RPG mode comes out, got my vote!

**Comment by rick594** [comment]
Yes,  leaning would definitely be a useful feature in the game I'm making at the moment.
